### DISCUSSION TITLE
Listbox (Select) doesn't allow changing value through typing search string in closed state

### DISCUSSION THREAD

**Discussion by luca-rath** [tailwindlabs/headlessui#922] [Ideas]
The default html select element allows the user to just start typing, when the select is focused but the menu is closed and it will automatically select the best matching option. I think it would be nice, if the Listbox component would behave the same. To give a real world example, I mostly love this functionality in a country selection, because there I always know the available options without opening the menu to see what's there

**Comment by grexi** [comment]
I'll miss this feature as well (starting to search through options without opening the ListBox first).

Ideally, an option to allow all navigation from this list: https://headlessui.com/react/listbox#keyboard-interaction
in case "when listbox is focused" and not only "when listbox is open".
